tests/infra_test.go

```diff
@@ -689,7 +689,11 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 			return nodeName
 		}
 
-		tests.DeprecatedBeforeAll(func() {
+		BeforeEach(func() {
```

Comment:
Can you please explain in the commit message the motivation fir this change and how it is related to the PR?